packages/backend/src/functions/httpTrigger.ts

```diff
@@ -187,9 +194,63 @@ async function postProvenance(request: HttpRequest, context: InvocationContext):
         const provRecord: ProvenanceRecord = { record, attachments };
         const data = new TextEncoder().encode(JSON.stringify(provRecord));
         const recordID = await upload(containerClient, deviceKey, data, "prov", "application/json", timestamp);
-        return { jsonBody: { record: recordID, attachments } };
+      return {
+        headers: { "Access-Control-Allow-Origin" : "*"},
+        jsonBody: { record: recordID, attachments } };
     }
 }
+// blobNames typically look like: 'gosqas/63f4b781c0688d83d40908ff368fefa6a2fa4cd470216fd83b3d7d4c642578c0/prov/1a771caa4b15a45ae97b13d7a336e1e9c9ec1c91c70f1dc8f7749440c0af8114'

```

Comment:
remove `typically`. Blob storage isn't exactly like a file system, but blob storage names look like file paths. The device ID *must* be the first segment of the path under `gosqas/` for the system to work

Reply:
This has been removed as requested.